docs/eventing/samples/helloworld/helloworld-python/README.md

```diff
@@ -97,7 +97,15 @@ cd knative-docs/docs/eventing/samples/helloworld/helloworld-python
       labels:
            eventing.knative.dev/injection: enabled
     ---
-    # Helloworld-python app deploment
+    # A default broker
+    apiVersion: eventing.knative.dev/v1
+    kind: Broker
+    metadata:
+      name: default
+      namespace: knative-samples
```

Comment:
We also need to add a kind of broker we want to create. So, something like:
```
name: default
namespace: knative-samples
annotations:
  eventing.knative.dev/broker.class: MTChannelBasedBroker
```
And maybe add a note saying that if you are using different kinds of brokers you might have to change this?

Reply:
done! PTAL